org.openhab.binding.zigbee.firmware/src/main/java/org/openhab/binding/zigbee/firmware/internal/DirectoryFileEntry.java

```diff
@@ -0,0 +1,189 @@
+/**
+ * Copyright (c) 2018-2021 by Z-Smart Systems Ltd.
```

Comment:
Could you please use the standard openHAB license header instead (on all new classes)?